src/common/stdin.cljs

```diff
@@ -16,5 +16,5 @@
   (.on stdin "end"
        (fn []
          ; (print "END EVENT!!!!")
-         (swap! stdinput #(->> % count dec (subs % 0)))
+         (swap! stdinput #(.trim %))
```

Comment:
Cool!

Reply:
Why not str/trim?

Reply:
have to check what that does - this was first thing that came to mind. most likely str/trim is just wrapping the method from the prototype, so ends up being the same. I'll look!

Reply:
str/trim does some normalisation for browser inconsistencies and some obscure google thing about being a trusted site - without digging into what that means for CLJS, it's probably better to use the provided library function, so have updated to use that instead.